src/CMakeLists.txt

```diff
@@ -4,14 +4,15 @@ set (CMAKE_CXX_STANDARD 17)
 if (DEFINED ENV{RELIC_MAIN})
   set(RELIC_GIT_TAG "origin/main")
 else ()
-  set(RELIC_GIT_TAG "b7b2266a0e4ee6f628f61d3ab638f524a18b52f1")
+  # This is currently anchored to upstream b7b2266a0e4ee6f628f61d3ab638f524a18b52f1
+  set(RELIC_GIT_TAG "d20d97c046da2c1689916569e9895b48a4693624")
 endif ()
 
 message(STATUS "Relic will be built from: ${RELIC_GIT_TAG}")
 
 FetchContent_Declare(
   relic
-  GIT_REPOSITORY https://github.com/relic-toolkit/relic.git
+  GIT_REPOSITORY https://github.com/AmineKhaldi/relic.git
```

Comment:
Are you planning to stay with your forked version or is this a development artifact?

Reply:
It's intentional yes, as I was advised to fork while we try to upstream eventually. I sent this one upstream already (https://github.com/relic-toolkit/relic/pull/207) but more will come.

Reply:
Just wanted to confirm